nautobot/docs/installation/nautobot.md

```diff
@@ -85,13 +85,26 @@ $ echo $NAUTOBOT_ROOT
 
 ## Understanding the Virtual Environment
 
+!!! important
+    You may need to start a new terminal session to see the expected environment results below.
+
```

Comment:
This is no longer the case now that we revised the install docs. See: #216 

(Please remove this admonition)

Reply:
removed